src/SideMenu.tsx

```diff
@@ -40,14 +45,19 @@ export const SideMenu: React.FC = (props) => {
   return (
     <div className="sidemenu" ref={ref}>
       <button className="sidemenu__btn" onClick={handleSelectorClicked}>
-        {currentSideMenuItems.length > 0 && t(currentSideMenuItems[0].children)}
+        {currentSideMenuItems.length > 0 ? t(currentSideMenuItems[0].children) : t("orders")}
```

Comment:
will there always be a currentSideMenuItems[0] element, or could this break in your new assumption?

Reply:
I was trying to check 2 other pathnames, I think it is  a better idea to call the URL same as what we did for "addressUrl" and add it to the "sideMenuItems" arrays. What do you think?
Add : `const orderDetailsUrl = createOrderDetailsUrl()`
Update : `const sideMenuItems = [
    { to: accountUrl, children: 'my-account' },
    { to: addressUrl, children: 'addresses' },
    { to: ordersHistoryUrl, children: 'orders' }
{ to: orderDetailsUrl, children: 'orders' }
  ];`

and change he condition to : `<Link to={elem.to} className={`sidemenu__link ${location.pathname.includes(elem.to) ? '--selected' : ''}`} onClick={handleHideMenu}>{t(elem.children)}</Link>`